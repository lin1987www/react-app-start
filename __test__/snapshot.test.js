import React from 'react';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
        .create(<a href="http://www.facebook.com">Facebook</a>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});