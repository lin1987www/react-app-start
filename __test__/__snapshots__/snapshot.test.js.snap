// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello World renders correctly 1`] = `
<h1>
  Hello, 
  John
</h1>
`;

exports[`renders correctly 1`] = `
<a
  href="http://www.facebook.com"
>
  Facebook
</a>
`;
